Src/Campus.Master.API/UI/jest.config.js

```diff
@@ -1,10 +1,16 @@
+const {pathsToModuleNameMapper} = require("ts-jest/utils");
```

Comment:
It's better to use the *import* directive instead of *require*, so the import can be performed asynchronously.

Reply:
Replaced !

Reply:
So, we face with the error such "SyntaxError: Cannot use import statement outside a module"

Reply:
Reverted

Reply:
I see we don't have the support of ES6 for now, so leave it as it is.